Cargo.toml

```diff
@@ -1,3 +1,10 @@
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+repository = "https://github.com/paritytech/polkadot-sdk.git"
+license = "GPL-3.0-only"
```

Comment:
The ones that are GPL3 with exception are not using the workspace license?

Reply:
Yes, its the same as before for those. But wen can also remove it and make it explicit for each crate :man_shrugging: